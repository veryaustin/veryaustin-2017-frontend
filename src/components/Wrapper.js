import styled from 'styled-components';

// Wrapper component for page width
const Wrapper = styled.div`
  width: 960px;
  max-width: 90%;
  margin: 0 auto;
`;

export default Wrapper;