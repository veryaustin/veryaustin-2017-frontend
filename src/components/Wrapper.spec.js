import React from 'react';
import {render} from 'enzyme';
import {renderToJson} from 'enzyme-to-json';
import Wrapper from './Wrapper';

describe('Wrapper', () => {
  it('renders the Wrapper component', () => {
    const wrapper = render(<Wrapper />);
    expect(renderToJson(wrapper)).toMatchSnapshot();
  });
});