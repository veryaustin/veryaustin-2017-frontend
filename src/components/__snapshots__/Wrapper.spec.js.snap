exports[`Wrapper renders the Wrapper component 1`] = `
<div
  class="guUdxq" />
`;
